EST.MIT.InvoiceImporter.Function/Startup.cs

```diff
@@ -63,6 +64,16 @@ public override void Configure(IFunctionsHostBuilder builder)
             }
         });
 
+
+        builder.Services.AddSingleton<IEventQueueService>(_ =>
+        {
```

Comment:
You will need to handle Managed Identity if a value of 'managedidentity' is supplied as config for QueueConnectionString:Credential by creating the client with DefaultAzureCredential(), in the same way as for Blob and Table clients. Use the IsManagedIdentity() method to determine if the config is for MID. If it's not, then use the raw connection string to create the client.